src/plugins/condition/components/Criterion.vue

```diff
@@ -52,7 +52,7 @@
                 <input v-model="criterion.input[inputIndex]"
                        class="c-cdef__control__input"
                        type="text"
-                       @blur="persist"
+                       @change="persist"
```

Comment:
This is going to create a whole lot of calls to persist the object. Was blur causing issues?